packages/inter-protocol/test/price/snapshots/test-fluxAggregator.js.md

```diff
@@ -0,0 +1,45 @@
+# Snapshot report for `test/price/test-fluxAggregator.js`
+
+The actual snapshot is saved in `test-fluxAggregator.js.snap`.
+
+Generated by [AVA](https://avajs.dev).
+
+## basic, with snapshot
+
+> Under "published", the "psm" node is delegated to
+> a tree of PSM contract instances, which publish data as follows.
+> 
+> See also board marshalling conventions (_to appear_).
+> 
+
+    [
+      [
+        'published.priceAggregator.LINK-USD_price_feed.latestRound',
+        {
+          roundId: 3n,
+          startedAt: 5n,
```

Comment:
i suppose this will change to a time record in #5799 so that we can keep from exposing the whole timer below